src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+            'Export Tags'                         : 'Exporter les étiquettes',
+            'FolderLabel'                         : 'Dossier',
+            'TagLabels'                           : 'Étiquette',
+            'ParentLabel'                         : 'Parent',
+            'FolderId'                            : 'Id du dossier',
+            'TagIds'                              : 'Ids des étiquettes',
+            'ParentId'                            : 'Id du parent',
+            'Empty'                               : 'Vide',
+            'Add Header Line'                     : 'Ajouter un en-tête',
+            'Export passwords shared with me'     : 'Exporter les mots de passe partagé avec moi',
+            'Don\'t edit passwords shared with me': 'Ne pas éditer les mots de passe partagé avec moi',
+            'Run Export'                          : 'Lancer l’export',
+            'Export'                              : 'Export',
+            'Waiting...'                          : 'En attente…',
+            'Export error'                        : 'Erreur lors de l’export',
+            'Download {format}'                   : 'Télécharger le ficher {format}',
+            'Nothing to export'                   : 'Rien à exporter',
+            'There is no data to export'          : 'Il n’y a aucune donnée à exporter',
+            'Restore or import'                   : 'Restauration/Import',
+            'Passwords CSV'                       : 'CSV des mots de passe',
+            'Folder CSV'                          : 'CSV des dossiers',
+            'Tags CSV'                            : 'CSV des étiquettes',
+            'Custom CSV'                          : 'CSV personnalisé',
+            'Select File'                         : 'Sélectionner un ficher',
+            'Select Options'                      : 'Choisissez les options',
+            'Conflict handling'                   : 'Gestion des conflits',
+            'Skip if same revision'               : 'Ignorer si n° de version identique',
+            'Skip always'                         : 'Toujours ignorer',
+            'Overwrite existing'                  : 'Écraser l’existant',
+            'Merge with existing'                 : 'Fusionner avec l’existant',
+            'Create new entry'                    : 'Créer une nouvelle entrée',
+            'For encrypted backups'               : 'Nécessaire pour les sauvegardes chiffrées',
+            'CSV Options'                         : 'Options CSV',
+            'Database'                            : 'Base de donnée',
+            'Quote Character'                     : 'Séparateur de chaîne de caractères',
+            'Escape Character'                    : 'Caractère d’échappement',
+            'Field delimiter'                     : 'Séparateur de champ',
+            'Field Delimiter'                     : 'Séparateur de champ',
```

Comment:
Why is this field duplicated ?
I do see that the case is different in the second one, but isn't it a mistake ?